internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,70 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
+	excludeDirs := []string{".git", ".zeabur"}
+	err = deleteFilesInDirectory(excludeFiles, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete files in directory: %w", err)
+	}
+	err = deleteDirectoriesInDirectory(excludeDirs, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete directories in directory: %w", err)
+	}
+
+	return nil
+}
+
+func deleteFilesInDirectory(deleteFiles []string, path string) error {
+	// walk through the directory
+	err := filepath.Walk(path, func(filePath string, fileInfo os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !fileInfo.IsDir() {
+			fileName := fileInfo.Name()
+
+			for _, targetFile := range deleteFiles {
+				if fileName == targetFile {
+					filePath := filepath.Join(path, fileName)
+
+					err := os.Remove(filePath)
+					if err != nil {
+						return err
+					}
+				}
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
 
+func deleteDirectoriesInDirectory(deleteDirs []string, path string) error {
```

Comment:
The behaviour of this function is pretty different to `deleteFilesInDirectory` but has the basically same kind of name. Would it better to rename `deleteFilesInDirectory` to `deleteFilesRecursively` while renaming `deleteDirectoriesInDirectory` to `deleteDirectory`?

By the way, just `err = os.RemoveAll("./<dir>")` and check `os.IsNotExist(err)` is enough – we don't really need to ReadDir.